beacon-chain/db/filesystem/data_column.go

```diff
@@ -0,0 +1,1014 @@
+package filesystem
+
+import (
+	"context"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/OffchainLabs/prysm/v6/async"
+	"github.com/OffchainLabs/prysm/v6/async/event"
+	"github.com/OffchainLabs/prysm/v6/beacon-chain/verification"
+	fieldparams "github.com/OffchainLabs/prysm/v6/config/fieldparams"
+	"github.com/OffchainLabs/prysm/v6/config/params"
+	"github.com/OffchainLabs/prysm/v6/consensus-types/blocks"
+	"github.com/OffchainLabs/prysm/v6/consensus-types/primitives"
+	"github.com/OffchainLabs/prysm/v6/io/file"
+	"github.com/OffchainLabs/prysm/v6/time/slots"
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+)
+
+const (
+	version                            = 0x01
+	versionOffset                      = 0                           // bytes
+	versionSize                        = 1                           // bytes
+	maxSszEncodedDataColumnSidecarSize = 536_870_912                 // 2**(4*8) / 8 bytes
```

Comment:
not used

Reply:
Woow, strange the linter is not complaining about not used private consts.
Thanks!

Reply:
Fixed in d3f949599dd4f1aa514daefbe2c708c099f2b81a.